rich/_log_render.py

```diff
@@ -60,11 +63,26 @@ def __call__(
                 log_time_display = time_format(log_time)
             else:
                 log_time_display = Text(log_time.strftime(time_format))
-            if log_time_display == self._last_time and self.omit_repeated_times:
-                row.append(Text(" " * len(log_time_display)))
+
+            if self.omit_repeated_times:
```

Comment:
I have a feeling this code can be simplified. There are two many levels of indentation and duplicated lines of code. Can you do another pass on this?